### PR TITLE
Release 0.11

### PR DESCRIPTION
**Pull request by aleksuss**
## Overview

Release 0.11

### Definition of Done

- [ ] There are no TODOs left in the merged code
- [ ] Change is covered by automated tests
- [ ] Benchmark results are attached (if applicable)
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

[coding guidelines]: https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions

### CODE REVIEW

File: examples/cryptocurrency-advanced/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Cryptocurrency Advanced: Service with Data Proofs
+    # Cryptocurrency Advanced: Service with Data Proofs
```
Comment (ozkriff) [inline examples/cryptocurrency-advanced/README.md]: this indent looks wrong

File: services/configuration/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "exonum-configuration"
-version = "0.10.1"
+version = "0.11.0"
 edition = "2018"
```
Comment (viktorbezhenar) [inline services/configuration/Cargo.toml]: Maybe 2019?

Comment (aleksuss) [reply services/configuration/Cargo.toml]: Not yet 😉

File: testkit/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "exonum-testkit"
-version = "0.10.1"
+version = "0.11.0"
 edition = "2018"
```
Comment (viktorbezhenar) [inline testkit/Cargo.toml]: and here 2019 too

**Pull request merged**